src/routes/users.js

```diff
@@ -17,12 +17,9 @@ router.get("/", function (req, res, next) {
     res.json(users);
   });
 
-  // db,User,finOne({ where:  {email:'Tan@techladies' }}}
 });
 
 router.get("/:surname", (req, res) => {
```

Comment:
Is this function being used? If not, can remove, otherwise better to put the auth.required on this route specifically- otherwise anyone can find out someone’s email address by only knowing their surname.

Reply:
Oh its not used, will remove it then